contracts/multiplesend/MultiSendFixedAmount.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+contract MultiSendFixedAmount {
+    function multiSend(address[] memory recipients, uint256 amount) external payable {
+        require(msg.value >= recipients.length * amount, "Not enough gas provided.");
+
+        for (uint256 i = 0; i < recipients.length; i++) {
+            (bool success, ) = recipients[i].call{value: amount}("");
```

Comment:
Use `transfer` instead of `call`, call can be a big security issue and consume all gas.

Reply:
have change transfer